src/makeCalculator.js

```diff
@@ -5,6 +5,41 @@
  */
 function makeCalculator() {
   // write code here
+  return {
+    result: 0,
+    add(calculator, number) {
+      calculator.result += number;
+
+      return calculator;
+    },
+    subtract(calculator, number) {
+      calculator.result -= number;
+
+      return calculator;
+    },
+    multiply(calculator, number) {
+      calculator.result *= number;
+
+      return calculator;
+    },
+    divide(calculator, number) {
+      if (number !== 0) {
```

Comment:
[CODE STYLE] - Instead of explicitly checking for division by zero, you should assume that the user will not divide by zero or handle the error outside of this method. The calculator methods should be kept simple and focused on their core functionality.